_tutorials-javascript/blog_feed.md

```diff
@@ -0,0 +1,126 @@
+
+Open your favorite text editor or IDE (atom, sublimetext, text edit, or even notepad).
+
+#### [index.html](https://github.com/steemit/devportal-tutorials-js/blob/master/tutorials/01_blog_feed/public/index.html)
+
+> Make a basic html file that contains the structure for the javascript to populate with results returned by the api. Basic bootstrap styling is used.
+
+#### [app.js](https://github.com/steemit/devportal-tutorials-js/blob/master/tutorials/01_blog_feed/public/app.js)
+
+> A seperate javascript file exists with the application to query the API and populate the HTML document with the results returned. We are calling `fetchBlog()` function when body of the page is onload. 
+
+#### [index.js](https://github.com/steemit/devportal-tutorials-js/blob/master/tutorials/01_blog_feed/index.js)
+
+> index.js is a basic javascript file that loads a `koa` based webserver serving the tutorial so you can access it via a browser
+
+#### fetchBlock
+
+> **fetchBlog** function creates simple query object where tag is set to account name on Steem and limit is set to number of posts being pulled from that account. By using `client.database.getDiscussions` dsteem function we are able to query user's blog posts and reformat them into list of posts.
+> Each blog post has `json_metadata` which holds meta information in post, using that we are able to extract first image from post and use it as thumbnail, post author and created information is also formated and displayed.
+
+
+``` javascript
+var query = {
+  tag: 'steemitblog', // This tag is used to filter the results by a specific post tag
+  limit: 5 // This limit allows us to limit the overall results returned to 5
+};
+```
+
+[Tag](/glossary/#Tags) - this tag is used to limit the types of posts that are returned. Each post is assigned multiple tags appon creation and by doing this allows you to filter what results are returned.
+
+`limit` - this limit allows us to limit the amount of records that are returned on a query. By not having a limit you could returned a large amount of data which might cause performance issues. By limiting the amount of results you return a smaller subset of data that you can work with.
+
+Place **script line** and **fetchBlog** function into `<head>` and save the file as `user-blog.html` and load it in your web browser. That's it, congratulations on your first app!
+
+#### JSON Result
+
+> A sample of the JSON returned by the `fetchBlog()` function is set out below. Key items to note are the following
+
+* url - This allows you to link back to the original article on Steem
+* json_metadata - This contains custom specific information for the post. It is a JSON object which must be parsed using `JSON.parse` to access. For posts this contains the image URL that is stored online
+
+``` json
+{% include tutorials-javascript/blog_feed.json %}
+```
+
+{% github_sample /steemit/condenser/blob/4718864eeb0074902b3261b786263a15015313cf/src/app/RootRoute.js %}
```

Comment:
@roboza I'm getting an error when trying to run the server on this branch:
```
Liquid Exception: Liquid syntax error (line 60): Unknown tag 'github_sample' in /Users/donovanwalker/projects/steemit/devportal/_tutorials-javascript/blog_feed.md
jekyll 3.3.1 | Error:  Liquid syntax error (lin
```
Can you explain, or fix?